run.sh

```diff
@@ -19,14 +19,22 @@ cp corpus/isa_b/* deps/ckb-vm/fuzz/corpus/isa_b/
 
 cd deps/ckb-vm
 
+fuzz() {
+    if [ -f ./fuzz/fuzz_targets/$1.rs ]; then
```

Comment:
It will fail silently if *.rs is missing.

Reply:
This is expected because the release-0.24 branch does not include the `interpreter` fuzz target